backend/aws_helpers/dynamo_db_utils/dynamo_db_utils.py

```diff
@@ -0,0 +1,198 @@
+import boto3
+from backend.aws_helpers.dynamo_db_utils.constants import ALL_DYANMODB_TABLES
+import random
+from datetime import datetime
+from typing import Union
+
+dynamodb = boto3.resource("dynamodb")
```

Comment:
do we need to specify aws region? @farisdurrani @dwu359

Reply:
Doesn't seem so, I believe the region is fed through the AWS CLI info that also has the access key ID and secret access key